### PR TITLE
School admin pagination added. EF-18537

### PR DESCRIPTION
**Pull request by OwaisRaza1994**
Ticket: https://everfi.atlassian.net/browse/EF-18537

Description: 
Pagination added in school admins response

### CODE REVIEW

File: lib/clever-ruby/models/school_admins_response.rb [hunk 1]
```diff
@@ -15,20 +15,24 @@
...
       }
     end
 
     # Attribute type mapping.
     def self.swagger_types
       {
-        :'data' => :'Array<SchoolAdminResponse>'
+        :'data' => :'Array<SchoolAdminResponse>',
+        :'paging'=> :'Object',
+        :'links'=> :'Object'
```
Comment (stephenjlovell) [inline lib/clever-ruby/models/school_admins_response.rb:35]: A couple minor style notes: 

* the quotes are unnecessary unless there are special characters in the symbol
* prefer using the shorthand `{ foo: :bar }` over the hashrocket style where possible
* spacing will need to get fixed on line 34-35

**Review by stephenjlovell — APPROVED** [review]
@OwaisRaza1994 looks fine, I left some minor style suggestions to save time on CodeClimate later

File: lib/clever-ruby/models/school_admins_response.rb (hunk 1 above)
Comment (tbodnya) [reply lib/clever-ruby/models/school_admins_response.rb:35]: @OwaisRaza1994 , lets update it, per Steve, please! It's ok, we can merge it tomorrow.

**Review by benrfeinberg — APPROVED** [review]

File: lib/clever-ruby/models/school_admins_response.rb (hunk 1 above)
Comment (tbodnya) [reply lib/clever-ruby/models/school_admins_response.rb:35]: oh well, it's merged now :) 
you can add it later...

**Pull request merged**